crates/spirv-attrib/src/lib.rs

```diff
@@ -1,16 +1,34 @@
-#![cfg_attr(target_arch = "spirv", feature(register_attr), register_attr(spirv))]
+use core::iter::FromIterator;
+use core::iter::IntoIterator;
```

Comment:
Pretty sure these are both imported by default? Or at least `IntoIterator` should be.

Reply:
From isn't but yeah apparently Into is.